lib/screens/landing_page/landing_page.dart

```diff
@@ -279,4 +264,52 @@ class _LandingPageState extends State<LandingPage> {
       ),
     );
   }
+
+  Widget weatherBox(Weather _weather) {
+    return Row(
+      mainAxisAlignment: MainAxisAlignment.spaceEvenly,
+      children: [
+        Column(
+          children: [
+            Image.asset(
+              'assets/images/cloudy.png',
+              height: MediaQuery.of(context).size.height * 0.08,
+            ),
+            Text("${_weather.description}",
+                style: TextStyle(color: light, fontSize: 15)),
+          ],
+        ),
+        Column(
+          children: [
+            Text("${_weather.temp}C",
+                style: TextStyle(color: light, fontSize: 20)),
+            SizedBox(height: 10),
+            Text('${DateFormat.yMMMd().format(DateTime.now())}',
+                style: TextStyle(
+                  color: light,
+                )),
+          ],
+        )
+      ],
+    );
+  }
+
+  Future getCurrentWeather() async {
+    Weather weather;
+    String city = "Kampala";
```

Comment:
Can we send the gps coordinates of the current location of the user, that way we shall be giving the user custom weather reports.